services/r2ctl/main/main.go

```diff
@@ -145,8 +136,7 @@ func main() {
 
 	services := []server.Service{
 		health.NewService(instrumentOpts),
-		swagger.NewService(instrumentOpts, "r2/v1"),
-		r2.NewService(instrumentOpts, "r2/v1", kv.NewStore(kv.StoreConfig{})),
+		r2.NewService(instrumentOpts, "r2/v1", stub.NewStore(instrumentOpts)),
```

Comment:
do you mean to drop swagger here?

Reply:
Yeah, swagger is served as a static file out of /public. It doesn't need a service.

Reply:
cool